docs/use-cases/collect-sensor-data.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Collect and view sensor data from any machines"
+linkTitle: "Collect sensor data"
+weight: 29
+type: "docs"
+images: ["/services/icons/data-query.svg"]
+description: "Gather sensor data, sync it to the cloud, and query it from the Viam app."
+modulescript: true
+# SME: Devin Hilly
+---
+
+You can use the data management service to capture sensor data from any machine and sync that data to the cloud.
+Then, you can query it using {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} to obtain actionable insights or connect it to third-party visualization tools.
+
+For example, you can configure data capture for several sensors on one machine, or for serveral sensors across multiple machines, to report the ambient operating temperature.
+You can then run queries against that data to search for outliers or edge cases, to analyze how the ambient temperature affects your machines' operation.
+
+You can do all of this using the [Viam app](https://app.viam.com/) user interface. You will not need to write any code.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Gathering data on any machine and syncing it to the cloud](#gather-and-sync-data).
+1. [Querying data in the Viam app](#query-data-in-the-viam-app).
+
+{{< /alert >}}
+
+## Prerequisites
+
+{{% expand "A running machine connected to the Viam app. Click to see instructions." %}}
+
+{{% snippet "setup.md" %}}
+
+{{% /expand%}}
+
+{{% expand "At least one configured sensor. Click to see instructions." %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Then [find and add a sensor model](/components/sensor/) that supports your sensor.
+
+{{% /expand%}}
+
+## Gather and sync data
+
+{{< readfile "/static/include/how-to/gather-sync-sensor.md" >}}
+
+## Query data in the Viam app
```

Comment:
I'm a little confused as to why this is here. Shouldn't this page just be about data collection, and maybe this section should move to the "sensor-data-query.md" page?